doc/main/intro/limitations.rst

```diff
@@ -5,12 +5,22 @@ Known Limitations
 
 This page outlines the known limitations of oneTBB to help you better understand its capabilities. 
 
+Debug TBB In The SYCL Program
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Limitation:** The application may crash when using the Debug version of oneTBB in a SYCL program compiled with Intel(R) oneAPI DPC++/C++ Compiler. This happens because both ``tbb`` (Release version) and ``tbb_debug`` (Debug version) libraries load simultaneously, causing conflicts.
```

Comment:
May be it is better to rephrase it - "when linking with the Debug oneTBB library"? Or is it clear what is using the Debug version of oneTBB?

Reply:
I suppose that "using ... in a SYCL program" implies thate the application is using the Debug TBB.